package.json

```diff
@@ -29,7 +38,9 @@
     "test": "jest",
     "start": "webpack-dev-server --env",
     "build:dev": "webpack --env",
-    "build:prod": "webpack --env.production"    
+    "build:prod": "webpack --env.production",
+    "lint": "eslint ./src/**/*.jsx",
+    "prettier": "prettier --write src/**/*.jsx"
```

Comment:
Let's add one more command to just check code w/ prettier. Rename this one according to what it does.